testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -601,6 +601,23 @@ public void testTrinoReadingSparkIcebergTablePropertiesData(StorageFormat storag
         onSpark().executeQuery("DROP TABLE IF EXISTS " + sparkTableName);
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS, ICEBERG_REST, ICEBERG_JDBC})
+    public void testSparkReadingTrinoIcebergTablePropertiesData()
+    {
+        String baseTableName = toLowerCase("test_spark_reading_trino_iceberg_table_properties");
```

Comment:
toLowerCase is redundant. Please change to `"test_spark_reading_trino_iceberg_table_properties" + randomNameSuffix()`. Then, dropping table before table creation isn't required.